src/app/features/tasks/task-summary-table/task-summary-table.component.ts

```diff
@@ -40,9 +42,13 @@ export class TaskSummaryTableComponent {
   }
 
   toggleTaskDone(task: Task) {
-    task.isDone
-      ? this._taskService.setUnDone(task.id)
-      : this._taskService.setDone(task.id);
+    if (task.isDone) {
+      this._taskService.setUnDone(task.id);
+      this._issueService.reopenIssue(task);
```

Comment:
I think It's better to stay consistent and implement this via an effect. Have a look at the jira effects there is something similar to this.